core/src/main/java/com/scalar/db/transaction/jdbc/JdbcTransactionAdmin.java

```diff
@@ -0,0 +1,109 @@
+package com.scalar.db.transaction.jdbc;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.inject.Inject;
+import com.scalar.db.api.DistributedTransactionAdmin;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.storage.jdbc.JdbcAdmin;
+import com.scalar.db.storage.jdbc.JdbcConfig;
+import java.util.Map;
+import java.util.Set;
+import javax.annotation.concurrent.ThreadSafe;
+
+@ThreadSafe
+public class JdbcTransactionAdmin implements DistributedTransactionAdmin {
```

Comment:
`JdbcTransactionAdmin` is the other one of the implementations of  `DistributedTransactionAdmin`. This class just transfers the operations to `JdbcAdmin` actually.